File: UNOPS.PAO.ClientApp/src/app/features/internal/models/organization-unit-relationship.model.ts
import { OrganizationHierarchyModel } from '../../../models/organization-hierarchy.model';

export interface OrganizationUnitRelationshipModel {
  organizationHierarchyId: number;
  organizationHierarchy?: OrganizationHierarchyModel;
  entityId: number;
  entityType: string;
} 

File: UNOPS.PAO.Models/OrganizationUnitRelationshipModel.cs
namespace UNOPS.PAO.Models;

public class OrganizationUnitRelationshipModel
{
    public int OrganizationHierarchyId { get; set; }
    public OrganizationHierarchyModel? OrganizationHierarchy { get; set; }
    public int EntityId { get; set; }
    public string EntityType { get; set; }
} 